lib/inquisition/bundler/audit/runner.rb

```diff
@@ -1,9 +1,35 @@
+require 'bundler/audit/cli'
+require 'bundler/audit/scanner'
+
 module Inquisition
   module Bundler
     module Audit
       class Runner < ::Inquisition::Runner
+        RUNNER_NAME = 'Bundler-audit'.freeze
+        LOW_LEVEL = :low
+        attr_reader :issues
+
         def call
-          puts 'Inquisition::Bundler::Audit::Runner'
+          ::Bundler::Audit::CLI.start [:update]
```

Comment:
Maybe would be better to work with [`Database`](https://github.com/rubysec/bundler-audit/blob/master/lib/bundler/audit/database.rb) instead if `CLI`, what do you think?